src/js/internal/util/parse_args/index.js

```diff
@@ -397,7 +397,19 @@ To specify an option argument starting with a dash use ${example}.`;
       return result;
     };
 
-    exports.parseArgs = parseArgs;
+    exports.parseArgs = Object.assign(parseArgs, {
+      // needed by tests
+      internal: {
```

Comment:
This change to the exports to support tests bothers me. I'm going to spend some time trying to figure out a way we can avoid this.